db/migrate.go

```diff
@@ -2,17 +2,26 @@ package db
 
 import (
 	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
 	"sort"
 
 	"github.com/lib/pq"
 )
 
 // InitSchema initialises the database tables and functions
-//go:generate go-bindata -pkg db -o bindata.go sql/...
 func (pc *PostgresClient) InitSchema() (err error) {
-	names := AssetNames()
-	sort.Strings(names)
-
+	// WARNING: this won't work if the binary is shipped without the code.
```

Comment:
I don't think this is true. If someone wanted to, they could deploy this software with their own SQL dir.

Not suggesting we should do it, but hypothetically wouldn't it be possible to separate the SQL dir and ship with just binary + SQL?

Can we delete this line?

Reply:
👍 

The reasons were stronger when it was using `runtime.Caller` because it referred to absolute paths on the machine that built the binary, which could have been very confusing. I don't think it's worth keeping now.

Reply:
removed